assignments/closure.js

```diff
@@ -1,7 +1,17 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
-
-
+function house(){
```

Comment:
Really nice job here, this is a popular interview question and this would be an excellent answer!